public/data/robertusbagaskara.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "Robertus Bagaskara",
+  "type": "personal",
+  "bio": "Computer Science Student that Loving Data Science 😊",
+  "avatar": "https://github.com/robertusbagaskara.png",
+  "links": [
+    {
+      "name": "Check out my Github",
```

I recommend capitalizing `h`. What do you think?
```suggestion
      "name": "Check out my GitHub",
```